classes/class-wc-connect-nux.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+if ( ! class_exists( 'Jetpack_Install_Status' ) ) {
```

Comment:
These could also be class constants in the `WC_Connect_Nux` class, perhaps prefixed with `JETPACK_`.